backend/codes/migrations/0010_code_priority.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.1.7 on 2021-05-12 19:11
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('codes', '0009_auto_20210512_2001'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='code',
+            name='priority',
+            field=models.PositiveIntegerField(default=0, validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(10)]),
+        ),
+    ]
```

Comment:
Dla naszego projektu to nie jest super ważne - ale jak zrobiłeś wcześniej migracje a pozniej cos ci sie zmienilo to warto je wyczyscic i zrobic dodac migracje tylko to co trzeba